TestsCalculator/CalculatorPageTests.cs

```diff
@@ -74,8 +291,21 @@ public void PositiveSelectRadioBtn()
             Assert.AreEqual("5.56", interestField.GetAttribute("value"));
         }
 
+        /*
+        How to check that a radio button is "checked"?
+        [Test]
+        public void CheckDefaultRadioBtnOption()
+        {
+            //Arrange
+            IWebElement daysRadioBtn365 = driver.FindElement(By.Id("d365"));
+
+            //Assert
+            Assert.True(daysRadioBtn365.checked);
```

Comment:
As I remember, you can use 'Selected' property to check if the checkbox or radio-button is selected.

```suggestion
            Assert.True(daysRadioBtn365.Selected);
```